### PR TITLE
Lit 79

### PR DESCRIPTION
**Pull request by BigKahuna7385**

### CODE REVIEW

**Comment by lawand7** [comment]
have you noticed that ?
image

**Comment by BigKahuna7385** [comment]
Fixed the color

File: src/test/java/com/littleBeasts/AttackStepDefinitions.java [hunk 1]
```diff
@@ -45,16 +47,24 @@ public void thePlayerIsNotDead() {
...
-        Program.getGameLogic().returnToMainMenu();
-        Assert.assertEquals(Program.getGameLogic().getState(), GameState.MENU);
+        robotButtonPress(KeyEvent.VK_E);
+        robotButtonPress(KeyEvent.VK_B);
+        Assert.assertEquals(GameLogic.getState(), GameState.INGAME);
+        returnToMainMenu();
+        Assert.assertEquals(GameLogic.getState(), GameState.MENU);
+    }
+
+    private void returnToMainMenu() {
```
Comment (lawand7) [inline src/test/java/com/littleBeasts/AttackStepDefinitions.java:64]: very nice

**Review by lawand7 — APPROVED** [review]
looks very good

**Pull request merged**